### ISSUE TITLE
Missing argument in the  SpectralTAD_Par function

### ISSUE THREAD

**Issue by anastasiya-pendragon** [dozmorovlab/SpectralTAD#10]
Good afternoon,

I would like to report the small bug we've encountered when using SpectralTAD_Par.R. If the desired output is granges, it does not work properly because the parameter is not specified in this function call:

https://github.com/dozmorovlab/SpectralTAD/blob/42ecc9d1de7132d63008bfbed9137f574ce6c2f0/R/SpectralTAD_Par.R#L98

Simply adding grange = grange seemed to have fixed the issue. 
```
spec_fun(cont_mat = cont_list[[x]],
                                                             chr[[x]], levels = levels, qual_filter = qual_filter,
                                                             z_clust = z_clust, eigenvalues = eigenvalues, min_size = min_size,
                                                             window_size = window_size, grange = grange),
```
Thank you for your package and your time!

**Comment by mdozmorov** [comment]
Hi @anastasiya-pendragon, thank you for reporting this. You are correct, we overlooked this. Would you be willing to do a pull request? No problem if not, but please, let me know.

**Comment by anastasiya-pendragon** [comment]
Hey @mdozmorov, thank you for your quick reply. I will create a pull request, thank you!

**Comment by mdozmorov** [comment]
Closed in #11
